Squirrely-Ducks/bangazon-sprint-3
# Jb shopping cart

## DESCRIPTION
Given that a user is logged in and has an open order, they may clear their shopping cart.

## STEPS TO TEST
1. npm start
2. log in
3. if no orders, search a product and add to cart
4. in the nav, click the shopping cart
5. click clear shopping cart
6. should be redirected to user account

## REFERENCE
Does it reference a ticket?
#1
